build.sbt

```diff
@@ -608,12 +605,14 @@ lazy val commonSettings = Def.settings(
         "-opt-inline-from:<source>",
         "-opt-warnings",
         "-opt:l:inline",
+        "-opt:l:method",
         "-explaintypes"
       )
     case Some((2, 13)) =>
       Seq(
         "-Xlint:-byname-implicit",
-        "-explaintypes"
+        "-explaintypes",
+        "-opt:l:method"
```

Comment:
I noticed that we don't have method-local optimizations enabled. These are pretty safe and binary compatible so I enabled them. I can remove them if needed